Manual.md

```diff
@@ -132,6 +135,83 @@ the Void packages system.
    requires the package.
 
+<a id="cross_compilation"></a>
+### Cross compilation
+
+Void Linux places high emphasis on having packages being able to be
+cross-compiled.
+
+In the context of Void Linux cross compilation is compiling software from one
+CPU architecture to another, e.g. compile to aarch64 (64bits ARM) from x86_64
+(Intel/AMD 64bits) system.
+
+To help with cross compilation xbps-src employs a sizeable amount of complexity
+to make cross compiling as easy as running `./xbps-src -a <arch> pkg <pkgname>`
+for the user.
+
+<a id="cross_terminology"></a>
+#### Terminology
+
+The Void Linux cross compilation framework has specific terminology for various
+concepts that may or not match what other projects use.
+
+The following is a list of terminology used:
+
+- `Native`: A compilation that is from and to the same architecture. Compiling
+from x86_64 to x86_64 is native.
+
+- `Cross`: A compilation that is from one architecture to another. Compiling
+from x86_64 to aarch64 is cross.
+
+- `Host`: The machine in which we are compiling the program, taking the Cross
+example x86_64 would be the host.
+
+> NOTE: Most projects refer to this as 'build'
+
+- `Target`: The machine for which we are generating the program, taking the
+Cross example aarch64 would be the target.
+
+> NOTE: Most projects refer to this as 'host'
+
+<a id="cross_setup"></a>
+#### Cross setup
+
+Cross compiling requires some changes in the environment from variables being
+set to wrappers being made and packages being installed in specific locations.
+
+xbps-src sources common/cross-profiles/<arch>.sh which sets some important
+variables, among them `XBPS_TARGET_MACHINE` which is the name of the
+architecture according to xbps-src and `XBPS_CROSS_TRIPLET` which is the GNU
+triplet which can be acquired from running `gcc -dumpmachine`.
+
+xbps-src install packages from `makedepends` and `checkdepends` to
+`/usr/$XBPS_CROSS_TRIPLET`, that value is stored in the variable
+`XBPS_CROSS_BASE`.
+
+xbps-src creates wrapper for `pkg-config` that forces it to search dependencies
+against what is installed on `XBPS_CROSS_BASE`.
+
+xbps-src performs build-style specific initialization:
+
+for `gnu-configure` it passes the --build, --host, --with-sysroot,
+--with-libtool-sysroot.
+
+for `cmake` it creates a .cmake definitions file called
+`cross_$XBPS_CROSS_TRIPLET.cmake` which sets the CMAKE_CROSS_COMPILING to TRUE,
+switches to FIND_ROOT_PATH to `XBPS_CROSS_BASE`, sets the c and cpp compilers
+to the cross-compilers, it also sets CMAKE_SYSTEM_PROCESSOR to the cpu family
+of the target.
+
+for `go` it sets GOARCH to the cpu family being used, if arm is being used it
+also sets GOARM.
+
+for `meson` it writes a crossfile which contains definitions of the path of
+the binaries to be used, the CFLAGS, CXXFLAGS and LDFLAGS to be used, it also
+sets the cpu and cpu family of the target, that file passed on --cross-file,
```

Comment:
Not sure what you're trying to say with `sets the cpu`.

Also, put a full stop instead of a comma in front of `that file` .

IMO `The crossfile is then passed to meson via --cross-file.` sounds better.

Put a full stop after that sentence instead of a comma.